.github/tools/release_linux.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+set -e -x
+
+# Remove the now private ppa. This can be removed after the docker image removes the
+# pre-installed python packages from this ppa.
+rm -f /etc/apt/sources.list.d/jonathonf-ubuntu-python-3_6-xenial.list
+
+ln -sf /usr/bin/python3.5 /usr/bin/python3 # Py36 has issues with add-apt
+add-apt-repository -y ppa:deadsnakes/ppa
+
+apt-get -y -qq update
+
+apt-get -y -qq install python${PYTHON_VERSION}-dev --no-install-recommends
+ln -sf /usr/bin/python${PYTHON_VERSION} /usr/bin/python
+ln -sf /usr/bin/python${PYTHON_VERSION} /usr/bin/python3
+
+ls /usr/include/x86_64-linux-gnu/
+ls /dt7/usr/include/x86_64-linux-gnu/
+
+# This really shouldn't be required to get 3.7 builds working
+ln -sf /usr/include/x86_64-linux-gnu/python3.7m /dt7/usr/include/x86_64-linux-gnu/python3.7m || true
+ln -sf /usr/include/x86_64-linux-gnu/python3.7m /dt8/usr/include/x86_64-linux-gnu/python3.7m || true
```

Comment:
Not sure what's going on here, but it looks like those files are missing from the custom op container.
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/ci_build/Dockerfile.custom_op_ubuntu_16#L77-L83

In any case this makes the Python 3.7 wheels build.